.gitignore

```diff
@@ -0,0 +1,139 @@
+# custom
+test.md
```

Comment:
- [ ] Great job for adding this file on `.gitignore` but I think you need to create also a file called `test.md` and write inside it `This file should be ignored by git` so kindly create this file as required 💯💪🏻☺

![image](https://user-images.githubusercontent.com/58553711/197897891-2ac99356-b4d8-47d4-a8c8-4e40810db8d3.png)

Reply:
Hi! Thanks for your feedback @Reem-lab .  As this file is included in .gitignore, you can't see it in my repository. But it's already created in my local application. Please, check the following image showing the file:

![image](https://user-images.githubusercontent.com/35902623/197907913-51a25a69-aedd-4f9c-a71a-7d55d0a2d290.png)

Reply:
I acutally cloned your repo and I am not able to see anything 💯

![image](https://user-images.githubusercontent.com/58553711/197910086-dfbecb00-3bd9-4718-9dbc-d9cdb1c9d368.png)